WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -2,6 +2,22 @@
 class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
+  def self.bad_keys_by_type_by_locale
+    @bad_keys_by_type_by_locale ||= begin
+      bad_keys_by_type_by_locale = {}
```

Comment:
We could even get rid of the method, not sure if it's better, but I seeing your code I thought of `Hash.new` with a block.
```ruby
BAD_KEYS = Hash.new do |hash, locale|
  ref_english = Locale.new('en')
  (I18n.available_locales - [:en]).each do |locale|
    missing, unused, outdated = Locale.new(locale, true).compare_to(ref_english)
    hash[locale] = { missing: missing, unused: unused, outdated: outdated }
  end
end
```